pom.xml

```diff
@@ -603,7 +614,12 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.18.1</version>
         <configuration>
-          <argLine>-Duser.language=en -Duser.country=US -XX:-UseSplitVerifier</argLine>
+          <systemPropertyVariables>
+            <jacoco-agent.destfile>${basedir}/target/jacoco.exec</jacoco-agent.destfile>
+          </systemPropertyVariables>
+          <properties>
+            <argLine> -Duser.language=en -Duser.country=US -XX:-UseSplitVerifier</argLine>
+          </properties>
```

Comment:
@MEZk you misunderstood this statement - in context of this statement ``properties`` refers to project properties and not to http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#properties